cmd/bnsd/app/codec.proto

```diff
@@ -55,36 +57,102 @@ message Tx {
     aswap.CreateSwapMsg create_swap_msg = 70;
     aswap.ReleaseSwapMsg release_swap_msg = 71;
     aswap.ReturnSwapMsg return_swap_msg = 72;
+    gov.CreateProposalMsg create_proposal_msg = 73;
+    gov.DeleteProposalMsg delete_proposal_msg = 74;
+    gov.VoteMsg vote_msg = 75;
+    gov.TallyMsg tally_msg = 76;
+    gov.UpdateElectorateMsg update_electorate_msg = 77;
```

Comment:
Q: I don't think it is possible to support UpdateElectorateMsg and a gov proposal with UpdateElectorateMsg payload at the moment on the same entity due to the authN check in https://github.com/iov-one/weave/blob/master/x/gov/handler.go#L515.

Reply:
You are right. It can *either* be done via sig/multisig or via gov. Not both. We can reflect if that is needed, and if so, how to do so

However, the code here allows that to be set in the genesis file - eg. a devnet could use alex's sig to change it. The public testnet a governance contract.

Reply:
ja, you are right, there may be other use cases in CI as well. I was eager to limit the options here and not let people have wrong assumptions about the updates process.